_mdlp.pxd

```diff
@@ -1,21 +0,0 @@
-import numpy as np
```

Comment:
I'm still concerned that you put in extra changes in this PR to remove this file, and move its contents to `mdlp/_mdlp.pyx`. But overall, the changes are good, and you were able to remove some dead code here.

Reply:
Certainly your concern makes sense. If I were more familiar with github pr workflows I would be more aggressively bending to your request to separate things.